internal/guest/README.md

```diff
@@ -0,0 +1,4 @@
+# Background
+The packages in this directory comprise most of the code used to build the guest side agent for Linux Hyper-V containers on Windows (LCOW). The two binaries of importance that are built from this code are the [main guest agent](../../cmd/gcs/main.go) that facilitates communication between the host and guest and a [generic tools binary](../../cmd/gcstools/main.go) for additional functionality for device scenarios. The guest agent is designed to run inside a custom Linux OS for supporting Linux container payloads.
```

Comment:
I think we should put the last sentence first and possibly as its own paragraph to highlight what the purpose of this is immediately.

Reply:
Done!